kafka-observ-lib/config.libsonnet

```diff
@@ -1,16 +1,17 @@
 {
   local this = self,
   filteringSelector: 'job!=""',
-  groupLabels: ['kafka_cluster'],
+  groupLabels: ['cluster'],
   instanceLabels: ['instance'],
   uid: 'kafka',
-  dashboardNamePrefix: 'Kafka',
+  dashboardNamePrefix: '',
   dashboardTags: ['kafka'],
   alertKafkaLagTooHighThreshold: '100',
   alertKafkaLagTooHighSeverity: 'critical',
   metricsSource: 'prometheus',  //or grafanacloud
   //Can be regex:
   topicsIgnoreSelector: '__consumer_offsets',
+  zookeeperEnabled: false,
```

Comment:
Do we not want to enable Zookeeper by default?

Reply:
you are right, changing.